app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,53 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: %i(show edit update destroy)
+
+  def index
+    @questions = Question.all
+  end
+
+  def show
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def edit
+  end
+
+  def create
+    @question = Question.new(question_params)
+    if @question.save
+      redirect_to @question, notice: t('question_was_successfully_created')
+    else
+      render :new
+    end
+  end
+
+  def update
+    if @question.update(question_params)
+      redirect_to @question, notice: t('question_was_successfully_updated')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.